huvr_client/media.py

```diff
@@ -0,0 +1,87 @@
+from typing import TYPE_CHECKING
+
+import requests
+import os
+import uuid
+
+if TYPE_CHECKING:
+    from .client import HuvrClient
+
+
+def upload_inspection_media(
+    client: "HuvrClient",
+    filepath: str,
```

Comment:
Other data you can pass in: geo_point, heading (IF+ does this). So maybe allow the rest of the payload to be set.